packages/flet/lib/src/flet_backend.dart

```diff
@@ -271,9 +274,10 @@ class FletBackend extends ChangeNotifier {
   /// - [control]: The control for which the event is triggered.
   /// - [eventName]: The name of the event to trigger.
   /// - [eventData]: Optional data to pass along with the event.
+  /// - [force]: Do not check if there is a subscriber for this event.
   void triggerControlEvent(Control control, String eventName,
       [dynamic eventData]) {
-    if (!isLoading && control.getBool("on_$eventName", false)!) {
+    if (control.get("on_$eventName") == true) {
```

Comment:
why remove `isLoading` check?

Reply:
Good question! There is a buffer now to "collect" all messages while connecting. Previous design was "swallowing" any events and property updates during connect/re-connect and it couldn't be done reliably to "know" when it's a good time to start sending messages to Python. Imagine, connect started, but you are continuing to drag window or change it's size, or add new multi-views and change routing. Now all these events are enqueued and send in batch after connect.